lib/netuitived.rb

```diff
@@ -82,9 +82,15 @@ def start(foreground = false)
 
       # Create a proc to run netuitived
       runner = proc do
+        NetuitiveD::NetuitiveLogger.log.debug 'starting scheduler'
         NetuitiveD::Scheduler.startSchedule
+        NetuitiveD::NetuitiveLogger.log.debug 'starting drb service'
+        begin
         DRb.start_service(server_uri, front_object)
```

Comment:
Spacing is off on these, makes it look like there are duplicate `end` statements.